Automattic/jetpack
# Payment blocks: Add "is_editable" property in the Membership API endpoints

Add support for the new  "public" property introduced in Memberships API.





Fixes https://github.com/Automattic/wp-calypso/issues/61908

#### Changes proposed in this Pull Request:

* Expose the "is_editable" property for Membership subscriptions

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Should be tested in context of https://github.com/Automattic/jetpack/pull/23669
* Should be tested in context of https://github.com/Automattic/jetpack/pull/23590
* Should be tested in context of https://github.com/Automattic/wp-calypso/pull/62208
